add a flake

stites opened the pull request:
this probably needs some documentation, but with this flake you no longer have to pull down the repo. You can just invoke the following:

```
❯ nix build --no-write-lock-file 'github:stites/nixos-aarch64-images/flakeit#rockPro64'
```

I didn't include a lock file because I didn't think it was needed.

```
❯ nix flake show
git+file:///home/stites/git/nixos-aarch64-images
└───packages
    └───x86_64-linux
        ├───aarch64Image: package 'aarch64-image'
        ├───pinebookPro: package 'image'
        ├───roc-pc-rk3399: package 'image'
        ├───rock64: package 'image'
        └───rockPro64: package 'image'
```

stites committed:
add flake

stites commented on flake.nix:
```diff
@@ -0,0 +1,22 @@
+          aarch64imgs = pkgs.callPackage ./. { };
+        in {
+          inherit (aarch64imgs)
+            aarch64Image rock64 rockPro64 roc-pc-rk3399 pinebookPro;
```
If `aarch64Pkgs` wasn't an output of `default.nix` this could be reduced to the `pkgs.callPackage ./. { }`

Mic92 replied:
Don't know why I actually it is there.

Mic92 commented:
Looks good. Can you add `❯ nix build --no-write-lock-file 'github:stites/nixos-aarch64-images/flakeit#rockPro64'` to the README.md as well?

stites commented:
yup -- I'm quickly asking on discord if I can also swap out inputs via the CLI for something like:

```
nix build --no-write-lock-file --switch-input nixpkgs=nixos-unstable -- 'github:stites/nixos-aarch64-images/flakeit#rockPro64' 
```

Mic92 commented:
There is `--override-input nixpkgs github:nixos/nixpkgs/nixpkgs-unstable`

stites committed:
document flakes

stites commented:
@Mic92 all done

Mic92 committed:
simplify flake a bit

I don't see yet any use case beyond using this on any other cpu than
x86_64-linux...,  maybe in future when risc64 takes over :)

So let's keep it simple for now

Mic92 commented:
Thanks a lot!
